### PR TITLE
complete issue25 tx input output

### PR DESCRIPTION
**Pull request by kdbusiness121**
issue #25 is completed

### CODE REVIEW

File: src/services/lib/bitcoingreen-lib.js [hunk 1]
```diff
@@ -22,3 +24,14 @@ export const importAddressFromPrivateKey = privateKey => {
...
   return address;
 };
+
+export const setTransaction = (txUtxos, txUtxoValue, amount, receiveAddress, senderAddress) => {
+  const rawTransaction = new bitcoin.TransactionBuilder();
+  for (let i = 0; i < txUtxos.length; i += 1) {
+    rawTransaction.addInput(txUtxos[i].txId, txUtxos[i].vout);
+  }
+
+  const change = txUtxoValue - amount - config.FEE_AMOUNT;
```
Comment (0xweb3builder) [inline src/services/lib/bitcoingreen-lib.js:34]: FEE_AMOUNT is defined?

File: src/services/lib/bitcoingreen-lib.js [hunk 2]
```diff
@@ -22,3 +24,14 @@ export const importAddressFromPrivateKey = privateKey => {
...
+  rawTransaction.addOutput(receiveAddress, amount);
+  rawTransaction.addOutput(senderAddress, change);
```
Comment (0xweb3builder) [inline src/services/lib/bitcoingreen-lib.js:36]: need return rawTransaction object.

**Review by 0xweb3builder — CHANGES_REQUESTED** [review]

File: src/services/lib/bitcoingreen-lib.js (hunk 1 above)
Comment (kdbusiness121) [reply src/services/lib/bitcoingreen-lib.js:34]: following by issue25: fee - this amount is fixed to a constant FEE_AMOUNT that we define in the app

Comment (0xweb3builder) [reply src/services/lib/bitcoingreen-lib.js:34]: good

**Review by 0xweb3builder — APPROVED** [review]

**Pull request merged**